x-pack/plugins/security_solution/public/management/pages/blocklist/view/blocklist.tsx

```diff
@@ -48,52 +51,47 @@ const BLOCKLIST_PAGE_LABELS: ArtifactListPageProps['labels'] = {
   ),
   flyoutCreateSubmitSuccess: ({ name }) =>
     i18n.translate('xpack.securitySolution.blocklist.flyoutCreateSubmitSuccess', {
-      defaultMessage: '"{name}" has been added to your blocklist.', // FIXME: match this to design (needs count of items)
+      defaultMessage: '"{name}" has been added to your blocklist.',
       values: { name },
     }),
   flyoutEditSubmitSuccess: ({ name }) =>
     i18n.translate('xpack.securitySolution.blocklist.flyoutEditSubmitSuccess', {
       defaultMessage: '"{name}" has been updated.',
       values: { name },
     }),
-  flyoutDowngradedLicenseDocsInfo: () => {
-    return 'tbd...';
-    // FIXME: define docs link for license downgrade message. sample code below
-
-    // const { docLinks } = useKibana().services;
-    // return (
-    //   <FormattedMessage
-    //     id="some-id-1"
-    //     defaultMessage="For more information, see our {link}."
-    //     value={{
-    //       link: (
-    //         <EuiLink target="_blank" href={`${docLinks.links.securitySolution.eventFilters}`}>
-    //           {' '}
-    //           <FormattedMessage
-    //             id="dome-id-2"
-    //             defaultMessage="Event filters documentation"
-    //           />{' '}
-    //         </EuiLink>
-    //       ),
-    //     }}
-    //   />
-    // );
+  flyoutDowngradedLicenseDocsInfo: (
+    securitySolutionDocsLinks: DocLinks['securitySolution']
+  ): React.ReactNode => {
+    return (
+      <>
+        <FormattedMessage
+          id="xpack.securitySolution.blocklist.flyoutDowngradedLicenseDocsInfo"
+          defaultMessage="For more information, see our "
+        />
+        <EuiLink target="_blank" href={`${securitySolutionDocsLinks.blocklist}`}>
+          <FormattedMessage
+            id="xpack.securitySolution.blocklist.flyoutDowngradedLicenseDocsLink"
+            defaultMessage="Blocklist documentation"
+          />
+        </EuiLink>
+      </>
+    );
   },
   deleteActionSuccess: (itemName) =>
     i18n.translate('xpack.securitySolution.blocklist.deleteSuccess', {
       defaultMessage: '"{itemName}" has been removed from blocklist.',
       values: { itemName },
     }),
   emptyStateTitle: i18n.translate('xpack.securitySolution.blocklist.emptyStateTitle', {
-    defaultMessage: 'Add your first blocklist',
+    defaultMessage: 'Add your first blocklist entry',
+  }),
+  emptyStateInfo: i18n.translate('xpack.securitySolution.blocklist.emptyStateInfo', {
+    defaultMessage:
+      'The blocklist prevents selected applications from running on your hosts by extending the list of processes the Endpoint considers malicious.',
```

Comment:
My understanding was that both applications and files could be blocked, not just applications -- is this correct? Also, a few other tweaks for flow and consistency.
```suggestion
      'The blocklist prevents selected applications or files from running on your hosts, extending the list of processes that Endpoint Security considers malicious.',
```